lib/improver/nowcasting/optical_flow.py

```diff
@@ -344,6 +373,9 @@ def __init__(self, data_smoothing_method='box', iterations=100):
             iterations (int):
                 Number of iterations to perform in post-calculation smoothing.
                 The value for good convergence is 20 (Bowler et al. 2004).
+            metadata_dict (dict):
```

Comment:
Should probably note that at the moment the same metadata dictionary is used to amend both the u and v cubes in the same way.

Reply:
Added a comment.